apis/rbac/types.go

```diff
@@ -50,12 +67,39 @@ type RoleRef struct {
 	Name string `json:"name" protobuf:"bytes,2,opt,name=name"`
 }
 
-// A Group is a grouping of RoleRefs.
-type Group struct {
+// ClusterRoleBinding grants the permissions defined in a ClusterRole referenced
+// to a user or a set of users
+type ClusterRoleBinding struct {
+	meta.TypeMeta   `json:",inline"`
+	meta.ObjectMeta `json:"metadata" protobuf:"bytes,1,opt,name=metadata"`
+
+	// Subjects holds references to the objects the role applies to
+	Subjects []Subject `json:"subjects" protobuf:"bytes,2,rep,name=subjects"`
+
+	// RoleRef is the reference to a ClusterRole in the global namespace
+	RoleRef RoleRef `json:"roleRef" protobuf:"bytes,3,name=roleRef"`
+}
+
+// RoleBinding grants the permissions defined in a Role referenced to a user or
+// a set of users
+type RoleBinding struct {
 	meta.TypeMeta   `json:",inline"`
 	meta.ObjectMeta `json:"metadata" protobuf:"bytes,1,opt,name=metadata"`
 
-	// RoleRefs is the list of RoleRefs that make up the roles for this
-	// Group.
-	RoleRefs []RoleRef `json:"roleRefs" protobuf:"bytes,2,rep,name=roleRefs"`
+	// Subjects holds references to the objects the role applies to
+	Subjects []Subject `json:"subjects" protobuf:"bytes,2,rep,name=subjects"`
+
+	// RoleRef is the reference to a Role in the current namespace
+	RoleRef RoleRef `json:"roleRef" protobuf:"bytes,3,name=roleRef"`
+}
+
+// Subject contains a reference to the user identify a role binding applies to
```

Comment:
Not sure I understand the docstring here :(

Reply:
identity instead of identify?